### PR TITLE
Dwworks additions

### PR DESCRIPTION
**Pull request by jgarcows**
Creating the pull request so automation also has time to run on this.

### CODE REVIEW

File: Lab 0/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+OS X: https://storage.googleapis.com/kubernetes-release/release/v1.5.6/bin/darwin/amd64/kubectl
+Linux: https://storage.googleapis.com/kubernetes-release/release/v1.5.6/bin/linux/amd64/kubectl
+Windows: https://storage.googleapis.com/kubernetes-release/release/v1.5.6/bin/windows/amd64/kubectl.exe
+
+Tip: If you are using Windows, install the Kubernetes CLI in the same directory as the Bluemix CLI. This setup saves you some filepath changes when you run commands later.
+
+For OSX and Linux users, complete the following steps.
+
+Move the executable file to the `/usr/local/bin` directory with `mv /<path_to_file>/kubectl/usr/local/bin/kubectl` .
```
Comment (jgarcows) [inline Lab 0/README.md:53]: Space needed after kubectl on the mv command.  `mv /<path_to_file>/kubectl /usr/local/bin/kubectl`

File: Lab 0/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# Install the Containers Service Plugin
+
+To create Kubernetes clusters and manage worker nodes, install the IBM Bluemix Container Service plug-in with `bx plugin install container-service -r Bluemix`. The prefix for running commands by using the IBM Bluemix Container Service plug-in is `bx cs`.
+
+To verify that the plug-in is installed properly, run the following command:
+`bx plugin list`
+
+The IBM Bluemix Container Service plug-in is displayed in the results as `container-service`.
+
+Once the bluemix CLI and the containers-service plugin is installed, you can move on to the next phase of the lab.
```
Comment (jgarcows) [inline Lab 0/README.md:39]: This line should be removed, as it is confusing.  Kubectl and bx cr are still needed for Lab 1 (some readers might mistake this as meaning you can leave lab 0 now)

File: Lab 6/README.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
+# CI/DC with the IBM Bluemix Containers Service
```
Comment (linsun) [inline Lab 6/README.md:1]: Do you mean CI/CD?

File: Lab 4/README.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
+# Highly Available Deployments With the IBM Bluemix Containers Service
+
+The goal of this lab is to begin to understand how to deploy a highly available application. It's easier than many think, but can be expensive if deploying across multiple AZs. The example in this lab shows how to deploy an application across two worker nodes in the same AZ (a basic level of high availability, to explore the concepts).
+
+# Federated Kubernetes Cluster: Two Worker Nodes Running the Same Application
```
Comment (linsun) [inline Lab 4/README.md:5]: does armada support this?

File: Lab 8/README.md [hunk 5]
```diff
@@ -0,0 +1,267 @@
+# Microservices in an Orchestrated World
+
+
+In this final lab, you will go beyond simple containers, and understand how to manage Microservices and Service Oriented Architecture beyond Kubernetes tools. This Lab will introduce Istio as a way to manage and dissect container and cluster traffic. It will use nearly everything you have learned in the previous lab, so please complete the previous labs if you have not already.
+
+
+
+# Before you Begin: Install Istio
```
Comment (linsun) [inline Lab 8/README.md:8]: The istio portion is v0.1 based, need to be updated for 0.2.

**Review by linsun — CHANGES_REQUESTED** [review]

**Review by jkomg — APPROVED** [review]
This LGTM, I'd like to have it added in to start on a clean-ish slate with our edits.

**Comment by jkomg** [comment]
@jgarcows & @nfritze I'd like to merge this so we can get to work pouring through it, is there any reason not to? We'd rather work from two sources (ours and this) rather than yours + a heavy PR with a ton of content + ours. 

Thoughts?

**Comment by jgarcows** [comment]
I've been working on doing the merge, hopefully on Monday it will be merged.  Sorry for all the delay.

**Comment by jgarcows** [comment]
Merged.  I'm closing this pull request.  Please make new pull requests for any changes needed.

**Pull request closed without merging**